{"name": "Login with empty mobile number", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.SessionNotCreatedException: Message: session not created: This version of ChromeDriver only supports Chrome version 109\nCurrent browser version is 111.0.5563.64 with binary path /usr/bin/google-chrome\nStacktrace:\n#0 0x5615adc98303 <unknown>\n#1 0x5615ada6cd37 <unknown>\n#2 0x5615ada99a54 <unknown>\n#3 0x5615ada9480e <unknown>\n#4 0x5615ada91330 <unknown>\n#5 0x5615adad24a6 <unknown>\n#6 0x5615adac9753 <unknown>\n#7 0x5615ada9ca14 <unknown>\n#8 0x5615ada9db7e <unknown>\n#9 0x5615adce732e <unknown>\n#10 0x5615adceac0e <unknown>\n#11 0x5615adccd610 <unknown>\n#12 0x5615adcebc23 <unknown>\n#13 0x5615adcbf545 <unknown>\n#14 0x5615add0c6a8 <unknown>\n#15 0x5615add0c836 <unknown>\n#16 0x5615add27d13 <unknown>\n#17 0x7fb6cae67fd4 <unknown>\n\n", "trace": "  File \"/usr/local/lib/python3.10/dist-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/usr/local/lib/python3.10/dist-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"orangehrm/pom/features/steps/loginsteps.py\", line 10, in launchBrowser\n    b.browserSetUp(context, \"chrome\")\n  File \"/usr/local/google/home/sateeshg/git-pythonselenium/orangehrm/pom/browsersetup/browseractions.py\", line 18, in browserSetUp\n    self.driver = webdriver.Chrome(\"/usr/local/google/home/sateeshg/git-pythonselenium/orangehrm/drivers/chromedriver109\")\n  File \"/usr/local/google/home/sateeshg/.local/lib/python3.10/site-packages/selenium/webdriver/chrome/webdriver.py\", line 69, in __init__\n    super().__init__(DesiredCapabilities.CHROME['browserName'], \"goog\",\n  File \"/usr/local/google/home/sateeshg/.local/lib/python3.10/site-packages/selenium/webdriver/chromium/webdriver.py\", line 92, in __init__\n    super().__init__(\n  File \"/usr/local/google/home/sateeshg/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 270, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"/usr/local/google/home/sateeshg/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 363, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n  File \"/usr/local/google/home/sateeshg/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 428, in execute\n    self.error_handler.check_response(response)\n  File \"/usr/local/google/home/sateeshg/.local/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py\", line 243, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given Launch the browser", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.SessionNotCreatedException: Message: session not created: This version of ChromeDriver only supports Chrome version 109\nCurrent browser version is 111.0.5563.64 with binary path /usr/bin/google-chrome\nStacktrace:\n#0 0x5615adc98303 <unknown>\n#1 0x5615ada6cd37 <unknown>\n#2 0x5615ada99a54 <unknown>\n#3 0x5615ada9480e <unknown>\n#4 0x5615ada91330 <unknown>\n#5 0x5615adad24a6 <unknown>\n#6 0x5615adac9753 <unknown>\n#7 0x5615ada9ca14 <unknown>\n#8 0x5615ada9db7e <unknown>\n#9 0x5615adce732e <unknown>\n#10 0x5615adceac0e <unknown>\n#11 0x5615adccd610 <unknown>\n#12 0x5615adcebc23 <unknown>\n#13 0x5615adcbf545 <unknown>\n#14 0x5615add0c6a8 <unknown>\n#15 0x5615add0c836 <unknown>\n#16 0x5615add27d13 <unknown>\n#17 0x7fb6cae67fd4 <unknown>\n\n", "trace": "  File \"/usr/local/lib/python3.10/dist-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/usr/local/lib/python3.10/dist-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"orangehrm/pom/features/steps/loginsteps.py\", line 10, in launchBrowser\n    b.browserSetUp(context, \"chrome\")\n  File \"/usr/local/google/home/sateeshg/git-pythonselenium/orangehrm/pom/browsersetup/browseractions.py\", line 18, in browserSetUp\n    self.driver = webdriver.Chrome(\"/usr/local/google/home/sateeshg/git-pythonselenium/orangehrm/drivers/chromedriver109\")\n  File \"/usr/local/google/home/sateeshg/.local/lib/python3.10/site-packages/selenium/webdriver/chrome/webdriver.py\", line 69, in __init__\n    super().__init__(DesiredCapabilities.CHROME['browserName'], \"goog\",\n  File \"/usr/local/google/home/sateeshg/.local/lib/python3.10/site-packages/selenium/webdriver/chromium/webdriver.py\", line 92, in __init__\n    super().__init__(\n  File \"/usr/local/google/home/sateeshg/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 270, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"/usr/local/google/home/sateeshg/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 363, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n  File \"/usr/local/google/home/sateeshg/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 428, in execute\n    self.error_handler.check_response(response)\n  File \"/usr/local/google/home/sateeshg/.local/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py\", line 243, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1678338697112, "stop": 1678338701096}, {"name": "When Open the URL https://www.abhibus.com/ website", "status": "skipped", "start": 1678338701097, "stop": 1678338701097}, {"name": "Then Click on the Login link", "status": "skipped", "start": 1678338701097, "stop": 1678338701097}, {"name": "And Enter Mobile number is empty", "status": "skipped", "start": 1678338701097, "stop": 1678338701097}, {"name": "And Click the Login or Signup button", "status": "skipped", "start": 1678338701097, "stop": 1678338701097}, {"name": "And Error message is displayed when mobile number is empty", "status": "skipped", "start": 1678338701097, "stop": 1678338701097}, {"name": "Then Close the browser", "status": "skipped", "start": 1678338701097, "stop": 1678338701097}], "start": 1678338697111, "stop": 1678338701097, "uuid": "25c19514-6b63-41d5-b6e7-76796e29e6c7", "historyId": "d2fbfaa0110ba1ac568ee5770ea06b2b", "fullName": "validate the login feature: Login with empty mobile number", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "invalid_login"}, {"name": "feature", "value": "validate the login feature"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}